module/VuFind/tests/integration-tests/src/VuFindTest/Mink/OAuth2Test.php

```diff
@@ -73,11 +72,8 @@ public static function setUpBeforeClass(): void
      */
     public function setUp(): void
     {
-        // Give up if we're not running in CI:
-        if (!$this->continuousIntegrationRunning()) {
-            $this->markTestSkipped('Continuous integration not running.');
-            return;
-        }
+        parent::setUp();
```

Comment:
Thanks for catching and fixing all this redundancy. I'm going to see if I can cherry-pick d09789ca4401bc85ac178349094c004f16986513 to dev so we can benefit from the cleanup immediately and reduce the number of changed files in this PR.